packages/magento-product-configurable/plugins/ConfigurableProductPageMeta.tsx

```diff
@@ -0,0 +1,48 @@
+import type { ProductPageProps } from '@graphcommerce/magento-product'
+import type { PluginProps } from '@graphcommerce/next-config'
+import { useConfigurableOptionsSelection } from '../hooks'
+
+export const component = 'ProductPageMeta'
+export const exported = '@graphcommerce/magento-product'
+
+type ConfigurableProductShortDescriptionProps = ProductPageProps & {
+  index?: number
+}
+
+function ConfigurableProductShortDescription(
+  props: PluginProps<ConfigurableProductShortDescriptionProps>,
+) {
+  const {
+    Prev,
+    index = 0,
+    sku,
+    categories,
+    price_range,
+    media_gallery,
+    name,
+    meta_title,
+    meta_description,
+    url_key,
+    __typename,
+    ...rest
+  } = props
+  const { configured } = useConfigurableOptionsSelection({ url_key, index })
+
+  const configurableOption = configured?.configurable_product_options_selection?.variant
```

Comment:
Might something like this be an idea? `mergeDeep(props, variant)`?

Reply:
In general, I'm not even sure this is necessary. This information only matters when a page loads, not dynamically, right?

Reply:
Yes, it's only necessary when the page loads, but if your simple products contain other metadata, you need to retrieve this data from the `configured?.configurable_product_options_selection?.variant`.